gr-uhd/lib/usrp_sink_impl.cc

```diff
@@ -730,18 +749,22 @@ void usrp_sink_impl::async_event_loop()
         }
         if (metadata.event_code & md_t::EVENT_CODE_UNDERFLOW) {
             event_list = pmt::list_add(event_list, UNDERFLOW_KEY);
+            ++underflow_counter;
         }
         if (metadata.event_code & md_t::EVENT_CODE_UNDERFLOW_IN_PACKET) {
             event_list = pmt::list_add(event_list, UNDERFLOW_IN_PACKET_KEY);
```

Comment:
This clause should also increase underflow_counter, right?

Reply:
Indeed!